lib/trackler/problem.rb

```diff
@@ -7,81 +7,108 @@ class Problem
     def initialize(slug, root)
       @slug = slug
       @root = root
+      @xcommon_repository_url = 'https://github.com/exercism/x-common/blob/master'
     end
 
     def exists?
-      !!md && !!yaml
+      description_exists? && yaml_exists?
```

Comment:
If a project has no style guide of it's own I like to stick to the Ruby style guide that recommends avoiding the use of `!!`: https://github.com/bbatsov/ruby-style-guide#no-bang-bang

Do you not think that `description_exists?` is more meaningful than `!!md`?

See discussion below about literate programming philosophy.